frontend/src/components/Tokenomics/PaydayAccountReward.vue

```diff
@@ -60,6 +62,12 @@ type Props = {
 	data: FilteredSpecialEvent<PaydayAccountRewardSpecialEvent>
 	goToPage: (page: PageInfo) => (target: PaginationTarget) => void
 }
+const props = defineProps<Props>()
 
-defineProps<Props>()
+const showFinalization = computed(() => {
```

Comment:
I think it would be nicer and more expressive if this evaluated to a bool instead of a number.